src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.html

```diff
@@ -29,6 +29,26 @@
        [innerHtml]="'Only available for RBD images with <strong>fast-diff</strong> enabled'"></div>
 </ng-template>
 
+<ng-template #provisionedNotAvailableTooltipTpl
+             let-row="row">
+  <span *ngIf="row.disk_usage === null; else provisioned"
+        [ngbTooltip]="usageNotAvailableTooltipTpl"
+        placement="top"
+        i18n>N/A</span>
```

Comment:
We are showing an `Unavailable` in the hosts table when orchestrator is not available for the gather facts field. I think its better to be consistent and show Unavailable over here too? @avanthakkar WDYT?

Reply:
I think that's for another reason, here I've shown `N/A` following what's shown in details section as well (value=`N/A`  and the similar text tooltip)